hooks/tk-multi-loader2/tk-nuke_actions.py

```diff
@@ -275,7 +275,7 @@ def _find_sequence_range(self, path, sg_publish_data):
         :returns: None if no range could be determined, otherwise (min, max)
         """
         # find a template that matches the path:
-
+        '''
```

Comment:
Please add a comment as to why this block is being commented out with associated ticket number.